docs/src/main/asciidoc/user-guide/topics/applying-conditions-to-mappings.adoc

```diff
@@ -1,193 +1,177 @@
-
-+
-
-When you add a field name to the expression, the data mapper adds 
-that field to the mapping. For example, consider this conditional 
-expression: 
-
+When you add a field name to the expression, the data mapper adds
+that field to the mapping. For example, consider this conditional
+expression:
 +
 image:second-conditional-mapping-expression.png[if(ISEMPTY(lastName), firstName, orderId + phone)]
 
-+
 During execution, if the data mapper determines that the `lastName`
 field is empty, it maps only the `firstName` field to the target
-`customerName` field.  If the `lastName` field contains a value, that is, 
-it is not empty, the data mapper concatenates the values in the 
-source `orderId` and `phone` fields, and inserts the result in 
-the `customerName` field. 
-(This example shows how the logic works, but it is probably not a 
-useful example because when there is a value in the `lastName` field, 
-you most likely want the data mapper to simply perform the 
+`customerName` field.  If the `lastName` field contains a value, that is,
+it is not empty, the data mapper concatenates the values in the
+source `orderId` and `phone` fields, and inserts the result in
+the `customerName` field.
+(This example shows how the logic works, but it is probably not a
+useful example because when there is a value in the `lastName` field,
+you most likely want the data mapper to simply perform the
 mapping and not map some other value into the target.)
 
 +
-For this example, after you complete entering the expression, 
-the data mapping is: 
+For this example, after you complete entering the expression,
+the data mapping is:
 
 +
-image:second-conditional-mapping.png[lastName, firstName, orderId, phone are mapped to customerName]
+image:second-conditional-mapping.png[lastName firstName orderId phone are mapped to customerName]
 
 +
-In the conditional expression, if you remove a field name that is in 
-the mapping that the expression applies to, the data mapper removes that 
+In the conditional expression, if you remove a field name that is in
+the mapping that the expression applies to, the data mapper removes that
 field from the mapping. In other words, every field name in the mapping
-must be in the conditional expression. 
+must be in the conditional expression.
 
-. If mapping preview fields are not already visible, display them 
-by clicking image:EditorSettings.png[Editor settings] and then 
-*Show Mapping Preview*. 
+. If mapping preview fields are not already visible, display them
+by clicking image:ShowMappingPreview.png[Show mapping preview].
 
-. Enter sample data in the source preview input field(s) 
+. Enter sample data in the source preview input field(s)
 to ensure that the target field or target fields
-get(s) the correct value. 
+get(s) the correct value.
 
 . Optionally, apply transformations to one or more source or target fields
 that are in the mapping:
-.. In the *Mapping Details* panel, find the field that you want to apply 
```

Comment:
In the Mapping Details panel, find the field that you want to apply
- Select the bolt icon and select the transformations from the pull-down menu.
- ...